### PR TITLE
feat(database): enable connection pooling

### PR DESCRIPTION
**Pull request by Harasz**

### CODE REVIEW

**Comment by Harasz** [comment]
@jrhender @hejkerooo I've tested it and everything seem to be fine. I would like to test it more on dev environment and investigate a problems if occurs.

File: README.md [hunk 1]
```diff
@@ -115,24 +102,35 @@ this can be generated using the command.
...
 In this case you need to initialize commitizen locally  
 `$ commitizen init cz-conventional-changelog --save-dev --save-exact`
+
+### Connection pooling
+
+To calculate maximum number of connections use following formula:  
+`{DBInstanceClassMemory/12582880}` - for example t2.micro has 512 MB RAM so maximum connections will be `(512*1024*1024)/12582880 ~= 40,`
+
+Then divide maximum connections number by number of maximum amount of IAM Cache Server instances.  
+If you are going to host 4 instances of ICS then `DB_MAXIMUM_CONNECTION_POOL` should be `40/4=10`
```
Comment (jrhender) [inline README.md:136]: Where is this formula derived from?

**Review by jrhender — COMMENTED** [review]
Why the README format changes? Is this driven by your editor or prettier config perhaps?

File: README.md (hunk 1 above)
Comment (Harasz) [reply README.md:136]: Yeah, sory I did not notice that, will disable prettier for `md` files. For formula you need to ask @hejkerooo

Comment (jrhender) [reply README.md:144]: @hejkerooo could you comment on the rational for the formula please?

Comment (hejkerooo) [reply README.md:144]: Formula should be `LEAST({DBInstanceClassMemory/9531392}, 5000)` 

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html

Comment (jrhender) [reply README.md:144]: Thanks @hejkerooo . What do you think of this change @Harasz ?
```suggestion
To calculate maximum number of connections use following formula:  
`LEAST({DBInstanceClassMemory/9531392}, 5000)`. This is taken from the [AWS RDS "Maximum number of database connections" documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html).

Then divide maximum connections number by number of maximum amount of IAM Cache Server instances.  
If you are going to host 4 instances of ICS then `DB_MAXIMUM_CONNECTION_POOL` should be `40/4=10`
```

**Review by jrhender — APPROVED** [review]
Approved with my suggested README change

**Pull request merged**